clusters/app.ci/cert-manager/prow_certificate.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: Certificate
```

Comment:
Why do we manually create certs rather than annotating the ingress?

Reply:
Updated.